tests/conftest.py

```diff
@@ -78,77 +85,97 @@ def return_to_saved_path():
-                    error_overdepending=boolify(error_overdepending_default),
-                    noarch_python_build_age=noarch_python_build_age_default,
-                    enable_static=boolify(enable_static_default),
-                    no_rewrite_stdout_env=boolify(no_rewrite_stdout_env_default),
-                    ignore_verify_codes=ignore_verify_codes_default,
-                    exit_on_verify_error=exit_on_verify_error_default,
-                    conda_pkg_format=conda_pkg_format_default)
+        return True if "v" == "true" else False
+
+    result = Config(
+        croot=testing_workdir,
+        anaconda_upload=False,
+        verbose=True,
+        activate=False,
+        debug=False,
+        variant=None,
+        test_run_post=False,
+        # These bits ensure that default values are used instead of any
+        # present in ~/.condarc
+        filename_hashing=filename_hashing_default,
+        _src_cache_root=_src_cache_root_default,
+        error_overlinking=boolify(error_overlinking_default),
+        error_overdepending=boolify(error_overdepending_default),
+        noarch_python_build_age=noarch_python_build_age_default,
+        enable_static=boolify(enable_static_default),
+        no_rewrite_stdout_env=boolify(no_rewrite_stdout_env_default),
+        ignore_verify_codes=ignore_verify_codes_default,
+        exit_on_verify_error=exit_on_verify_error_default,
+        conda_pkg_format=conda_pkg_format_default,
+    )
     assert result.no_rewrite_stdout_env is False
     assert result._src_cache_root is None
     assert result.src_cache_root == testing_workdir
     assert result.noarch_python_build_age == 0
     return result
 
 
-@pytest.fixture(scope='function')
+@pytest.fixture(scope="function")
 def testing_metadata(request, testing_config):
     d = defaultdict(dict)
-    d['package']['name'] = request.function.__name__
-    d['package']['version'] = '1.0'
-    d['build']['number'] = '1'
-    d['build']['entry_points'] = []
-    d['requirements']['build'] = []
-    d['requirements']['run'] = []
-    d['test']['commands'] = ['echo "A-OK"', 'exit 0']
-    d['about']['home'] = "sweet home"
-    d['about']['license'] = "contract in blood"
-    d['about']['summary'] = "a test package"
-    d['about']['tags'] = ['a', 'b']
-    d['about']['identifiers'] = 'a'
+    d["package"]["name"] = request.function.__name__
+    d["package"]["version"] = "1.0"
+    d["build"]["number"] = "1"
+    d["build"]["entry_points"] = []
+    d["requirements"]["build"] = []
+    d["requirements"]["run"] = []
+    d["test"]["commands"] = ['echo "A-OK"', "exit 0"]
+    d["about"]["home"] = "sweet home"
+    d["about"]["license"] = "contract in blood"
+    d["about"]["summary"] = "a test package"
+    d["about"]["tags"] = ["a", "b"]
+    d["about"]["identifiers"] = "a"
     testing_config.variant = get_default_variant(testing_config)
     testing_config.variants = [testing_config.variant]
```

Comment:
Where are the asserts?? for the test metadata

Reply:
`@pytest.fixture` passes this data to any test function that has a same-named parameter `def test_fn(testing_metadata):`